### ISSUE TITLE
Training probelm

### ISSUE THREAD

**Issue by hejs9603** [EndlessSora/focal-frequency-loss#12]
The warning code is【C:\Users\PC\.conda\envs\paGAN\lib\site-packages\torch\autograd\__init__.py:173: UserWarning: Casting complex values to real discards the imaginary part (Triggered internally at  C:\cb\pytorch_1000000000000\work\aten\src\ATen\native\Copy.cpp:239.)
  Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass】
  
  Does it affect the quality of the generated images? Thank you!

**Comment by EndlessSora** [comment]
Thanks for you interest. Duplicate with https://github.com/EndlessSora/focal-frequency-loss/issues/10.

**Comment by hejs9603** [comment]
> Thanks for your interest. Duplicate with #10.

Ok，Thank you！
